third_party/silabs/siwx917_executable.gni

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) 2020 Project CHIP Authors
```

Comment:
I don't believe there is a reason for a duplicate here. This is what forms the binary and flashing package. I don't think there is anything different for the siwx917 (correct me if I am wrong).

What I mean when we discussed changing efr32_executable is to just rename it for all to `silabs_executable.gni`

Reply:
modified on the latest commit